src/components/ViewBusiness/ViewBusiness.jsx

```diff
@@ -0,0 +1,203 @@
+                  aria-label="Edit menu"
+                  icon={<EditIcon boxSize="6" />}
+                  mr={4}
+                />
+                <IconButton
+                  variant="ghost"
+                  colorScheme="gray"
+                  aria-label="Delete menu"
+                  icon={<DeleteIcon boxSize="6" />}
+                />
+              </Flex>
+            </Flex>
+          </CardHeader>
+          <CardBody>
+            <Flex alignItems="left">
+              <TableContainer>
+                <Table variant="simple">
+                  <Thead></Thead>
+                  <Tbody>
+                    <Tr>
+                      <Td>NAME</Td>
+                      <Td>{data.contact_name}</Td>
+                    </Tr>
+                    <Tr>
+                      <Td>EMAIL</Td>
+                      <Td>{data.primary_email}</Td>
+                    </Tr>
+                    <Tr>
+                      <Td>WEBSITE</Td>
+                      <Td>{data.website}</Td>
+                    </Tr>
+                    <Tr>
+                      <Td>LOCATION</Td>
+                      <Td>{data.street + ' ' + data.qb_city_state_zip}</Td>
+                    </Tr>
+                  </Tbody>
+                </Table>
+              </TableContainer>
+            </Flex>
+          </CardBody>
+          <Flex direction="row" align="center" w="full" mt={4}>
+            <Box mr={12}>
+              <b>DONATION FORM STATUS</b>
+              <HStack spacing={4} mt={15}>
+                <Tag size={'lg'} key={'md'} variant="solid" colorScheme="teal">
+                  <DownloadIcon ml={-1} mr={2} />
+                  <b>Download CSV</b>
+                </Tag>
+              </HStack>
+              <Text fontWeight={600}>Submitted on</Text>
+            </Box>
+
+            <Box>
+              <b>SUPPLY REQUEST</b>
+              <HStack spacing={4} mt={15}>
+                <Tag size={'lg'} key={'md'} variant="outline" colorScheme="gray">
+                  <CalendarIcon ml={-1} mr={2} />
+                  <b>View Request</b>
+                </Tag>
+              </HStack>
+              <Text fontWeight={600}>Requested on</Text>
+            </Box>
+          </Flex>
+          <CardFooter
+            justify="space-between"
+            flexWrap="wrap"
+            sx={{
+              '& > button': {
+                minW: '136px',
+              },
+            }}
+          ></CardFooter>
+        </Card>
+        <Flex justify="center" mt={8}>
+          <Card maxW="100%" h="auto" p={8}>
+            <Box mt={4} mb={4} px={4}>
+              <Text fontWeight="bold" fontSize="lg" mb={4}>
+                Donation Form History
+              </Text>
+            </Box>
+            <CardBody>
+              <Stack spacing={4} px={4}>
+                <Divider />
+                <Box>
+                  <Text pt="2" fontSize="md" mb={5}>
+                    1/12/2024
```

Comment:
you can get the actual dates if you run a GET by business_id on the donation router and use the timestamps on each of those and display them here